llvm/test/CodeGen/SPIRV/unnamed_global.ll

```diff
@@ -0,0 +1,15 @@
+; RUN: llc -O0 -mtriple=spirv32-unknown-unknown %s -o - | FileCheck %s
+
```

Comment:
Could spirv-val run line be added in this test? Also, please change the name to use dashes instead of underscores.

Reply:
Can we please address spirv-val usage separately? In this (and many other cases) it would result in validation error, unrelated to this particular PR. In this case, for example, it would be a complain to OpExecutionMode. It may be that the question of spirv-val deserves its own (one or more) issue.